github/release_notes/renderer.py

```diff
@@ -210,6 +210,9 @@ def _to_bullet_points(
         bullet_points = list()
         for rls_note_obj in rls_note_objs:
             for i, rls_note_line in enumerate(rls_note_obj.text.splitlines()):
+                rls_note_line = _.trim(rls_note_line)
+                if not rls_note_line:
```

Comment:
why not rather use map + filter? this would be more consistent with other code from release_notes